### PR TITLE
chore: setup changesets

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-2161]

### Description

Introduces changesets into Picasso monorepo. It also automatically appends PR body with warning to not forget about peerDependencies. I have tested it out on testing branch.

### How to test

you can check generated PR from changesets 👉 https://github.com/toptal/picasso/pull/2215

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

[FX-2161]: https://toptal-core.atlassian.net/browse/FX-2161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install Dependencies (from network)
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --ignore-optional
+
+      - name: Install Dependencies (from cache)
+        if: steps.yarn-cache.outputs.cache-hit == 'true'
+        run: yarn install --frozen-lockfile --ignore-optional --offline
```
Comment (dmaklygin1) [inline .github/workflows/release.yml:52]:
```suggestion
      - name: Check yarn cache
      uses: c-hive/gha-yarn-cache@v2

    - name: Install Dependencies
      shell: bash
      run: |
        yarn policies set-version
        yarn install --frozen-lockfile --ignore-optional

```

Comment (dmaklygin1) [reply .github/workflows/release.yml:52]: wdyt?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            for await (let pr of data) {
+              github.rest.issues.addLabels({
+                owner: 'toptal',
+                repo: 'picasso',
+                issue_number: pr.number,
+                labels: [
+                  'no-jira'
+                ]
+              })
+            }
```
Comment (dmaklygin1) [inline .github/workflows/release.yml:88]: https://github.com/changesets/action/issues/50#issuecomment-851522508
there is an alternative approach to set a label. wdyt?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+              const warningTodo = "\n- [ ] ⚠️ "
+              const message = "If major release, don't forget to check if peerDependencies needs to be also updated"
+              const appendedMessage = hr + warningTodo + message
+              const body = pr.body.includes('message') ? pr.body : pr.body + appendedMessage
+              github.rest.pulls.update({
+                owner: 'toptal',
+                repo: 'picasso',
+                pull_number: pr.number,
+                body: body,
+              })
```
Comment (dmaklygin1) [inline .github/workflows/release.yml:116]: and, if the previous approach works, we could reuse it for this action as well. This is an example of changing body: https://cli.github.com/manual/gh_pr_edit

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+          jenkins_user: ${{ env.TOPTAL_BOT_USERNAME }}
+          jenkins_token: ${{ env.TOPTAL_BOT_JENKINS_DEPLOYMENT_TOKEN }}
+          proxy: ${{ env.HTTP_PROXY }}
+          job_name: ${{ env.JENKINS_JOB_NAME }}
+          job_params: |
+            {
+              "COMMIT_ID": "${{ github.sha }}"
+            }
+          job_timeout: '3600'
+      # - name: Send a Slack notification on failure
```
Comment (teimurjan) [inline .github/workflows/release.yml:135]: Is this comment part intentional?

Comment (TomasSlama) [reply .github/workflows/release.yml:135]: It is not, thanks, forgot to comment it out after testing.

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+      - name: Setup Node.js 14.x
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 14.x
+
+      - name: Check yarn cache
+        uses: c-hive/gha-yarn-cache@v2
+
+      - name: Install Dependencies
+        shell: bash
```
Comment (dmaklygin1) [inline .github/workflows/release.yml:37]:
```suggestion

```
I guess it's not needed anymore

**Review by denieler — APPROVED** [review]
what do you think also about adding such a section to our contribution readme? https://github.com/toptal/davinci#release

**Review by dmaklygin1 — APPROVED** [review]
Good work!

**Review by OleksandrNechai — APPROVED** [review]

**Pull request merged**